v5/core/constants.go

```diff
@@ -45,6 +45,7 @@ const (
 	PROPNAME_CLIENT_ID        = "CLIENT_ID"
 	PROPNAME_CLIENT_SECRET    = "CLIENT_SECRET"
 	PROPNAME_SCOPE            = "SCOPE"
+	/* #nosec G101 */
```

Comment:
I hate to ask this but would it also work to use an inline "//-style" command at the end of the affected line?
If so, I think I'd prefer to just be consistent with the way in which we annotate code to avoid the false positives.